Update project-structure.md (#297)
nuejs/nue: packages/nuejs.org/docs/project-structure.md

# Project Structure
Nue projects consist of one or more _applications_. Each directory in the root of your project folder is a separate application with its own configuration, layout, styling and scripting.

[image.gridpaper]
  small: /img/application-dirs.png
  large: /img/application-dirs-big.png
  caption: Two new areas were added as the project grew up

Nue does not force you to adopt any fixed directory structure: there are no system folders and you can freely name your root-level application folders.

## Applications
Nue is designed for building two kinds of applications:

1. **Multi-page applications**. These are content-focused applications consisting of Markdown files. Good examples are documentation, blogging area or a feature tour. These apps are rendered server-side so that they can be easily consumed by search engines. The use of client-side JavaScript is optional.

> 3. [Single-page applications](single-page-applications.html) consist of an `index.html` file that serves all the HTML requests within the app. Good examples are admin dashboards, onboarding flows, surveys or login pages. The application is rendered on the client side with reactive components. These apps are usually hidden from search engines.

Reviewer: ```suggestion
2. [Single-page applications](single-page-applications.html) consist of an `index.html` file that serves all the HTML requests within the app. Good examples are admin dashboards, onboarding flows, surveys or login pages. The application is rendered on the client side with reactive components. These apps are usually hidden from search engines.
```
:)

exclude: [syntax-highlight, video]
```

This allows you to strip unneeded assets from the request and reduce the payload.

#### Example

> Here's a more complex example that give you an idea how dependencies are calculated:

Reviewer: ```suggestion
Here's a more complex example that can give you an idea how dependencies are calculated:
```
Or: `gives`

> Here's a more complex example that can give you an idea how dependencies are calculated:

Reviewer: ```suggestion
Here's a more complex example that gives you an idea how dependencies are calculated:
```

See https://github.com/nuejs/nue/commit/3fb8221bacdc9a84e57eb6eec8fc867ee1c98d9a#r145060044

### 404 file

> File named `404.md` in the root level acts as a target for non-existent files. You can use the `include` statement to customize its styling and behaviour.

Reviewer: ```suggestion
A file named `404.md` in the root level, acts as a target for non-existent files. You can use the `include` statement to customize its styling and behaviour.
```
